llvm/lib/Transforms/FuzzIntrospector/FuzzIntrospector.cpp

```diff
@@ -1317,6 +1335,52 @@ FuzzIntrospector::findReachables(BasicBlock *Src) {
   return AllReachables;
 }
 
+vector<StringRef> FuzzIntrospector::findReachableFuncs(BasicBlock *Src) {
```

Comment:
Could you add a comment about the function? In particular that this is local to a function and not program-wide reachability?